manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1829,21 +1839,48 @@ def _create_axis(
         axis.shift(-axis.number_to_point(self._origin_shift([axis.x_min, axis.x_max])))
         return axis
 
-    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+    def coords_to_point(self, *coords: Sequence[float] | np.ndarray) -> np.ndarray:
```

Comment:
According to the docstring that follows this, shouldn't there also be a `Sequence[Sequence[float]]` for better typing? Something of the sort:
```suggestion
    def coords_to_point(self, *coords: Sequence[float] | Sequence[Sequence[float]] | np.ndarray) -> np.ndarray:
```

Also, not sure about this, but wouldn't `np.ndarray` already be covered as a `Sequence` here, and could thus be removed?

Reply:
Yes, Sequence probably includes a `np.ndaray` but the problem is that a `Sequence` does not require the implementation of mathematical operators which are heavily used in the vectorized versions. So i wanted to make explicit that you can either pass a sequence containing floats or a corresponding ndarray which implements all the needed functionality.

Reply:
Got it, but the suggestion still holds valid right? The `Sequence[Sequence[float]]` one.

Reply:
A single point is `Sequence[float]` a list of points is `np.ndarray`. `Sequence[Sequence[float]]` would just duplicate `np.ndarray` but without the needed functionality 🤔 .

Reply:
i would be fine with changing it to `npt.NDarray[np.float64]` but that's why theres a typing discussion. We can probably delay the "correct" typing of this function.

Reply:
Wait nvm ... i used `np.asarray`... youre right i haven't looked at my own code for too long! YES YES!!! 🥳